packages/headless/src/app/engine.test.ts

```diff
@@ -90,4 +91,32 @@ describe('engine', () => {
     expect(await engine.renewAccessToken()).toBe('newToken');
     done();
   });
+
+  it("when name is specified in the config, the engine's store name is initialized with the config's value", () => {
+    options.configuration.name = 'myEngine';
+    const spy = jest.spyOn(Store, 'configureStore');
+    initEngine();
+
+    expect(spy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        name: options.configuration.name,
```

Comment:
```suggestion
        name: 'myEngine',
```

Defeats the test purpose a bit if you use `options.configuration.name` both to set the value and to compare with the expect